evaluation.py

```diff
@@ -97,7 +99,40 @@ def save_stacked_evaluation_heatmaps(self, h, X, y, stacked_predict_heatmaps_fil
         img_v_resize = self._vstack_images(heatmap_imgs) 
         
         cv2.imwrite(filename, img_v_resize) 
-    
+
+    # Resources for heatmaps to keypoints
+    # https://github.com/yuanyuanli85/Stacked_Hourglass_Network_Keras/blob/eddf0ae15715a88d7859847cfff5f5092b260ae1/src/eval/heatmap_process.py#L5
+    # https://github.com/david8862/tf-keras-stacked-hourglass-keypoint-detection/blob/56707252501c73b2bf2aac8fff3e22760fd47dca/hourglass/postprocess.py#L17
+   
+    ### Returns np array of predicted keypoints from one image's heatmaps
+    def heatmaps_to_keypoints(self, heatmaps, threshold=HM_TO_KP_THRESHOLD):
+        keypoints = list()
+        for i in range(NUM_COCO_KEYPOINTS):
+            hmap = heatmaps[:,:,i]
+            # Resize heatmap from Output DIM to Input DIM
+            resized_hmap = cv2.resize(hmap, INPUT_DIM, interpolation = cv2.INTER_AREA)
+            # Do a heatmap blur with gaussian_filter
+            resized_hmap = gaussian_filter(resized_hmap, HEATMAP_SIGMA)
+
+            # Get peak point (brightest area) in heatmap with 3x3 max filter
+            peaks = self._non_max_supression(resized_hmap, windowSize=3, threshold=HM_TO_KP_THRESHOLD)
```

Comment:
I think you might have missed using the parameter that's been passed in 
```suggestion
            peaks = self._non_max_supression(resized_hmap, windowSize=3, threshold=threshold)
```